to-do-list/src/components/List.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Todo from './Todo'
+
+// temporary static data
+const tasks = [
+  {id:1, task: "Dance a lot"},
+  {id:2, task: "Run a mile"},
+  {id:3, task: "Drink some soda"},
+  {id:4, task: "Task #4"},
+  {id:5, task: "Computers"},
+  {id:6, task: "Code"}
+]
+
+const TodoList = (props) => {
+  const todos = props.tasks
```

Comment:
would it be better to keep the naming consistent to either todos or tasks, since the props passed to TodoList is declared as tasks when TodoList is called later in List?